### ISSUE TITLE
Docker examples in README does not work

### ISSUE THREAD

**Issue by Mange** [jonmorehouse/dogstatsd-local#9]
The examples in the README doesn't really work as expected. The `/udp` option must be provided to the port specifications for it to work for me.

```
docker run --rm -p 8125:8125/udp jonmorehouse/dogstatsd-local -format raw
```

The first example also uses `8126:8126`, which also would not work since the daemon binds to `8125`.

**Comment by nkg447** [comment]
Yes it worked for me too. I guess this should be added in README.
